src/FileInfo.cpp

```diff
@@ -106,6 +79,55 @@ FileInfo GetFileInfo(const char *path)
 
     result.m_Size = stbuf.st_size;
 
+#elif defined(TUNDRA_WIN32)
+
+    std::wstring widePath(ToWideString(path));
+    //// To work around maximum path length limitations on Windows, we have to use the wide-character version of the API, with a special prefix
+    if (!ConvertToLongPath(&widePath))
+        goto Failure;
+
+    HANDLE hFile = CreateFileW(widePath.c_str(),
+                            GENERIC_READ,
+                            FILE_SHARE_READ |
+                            FILE_SHARE_WRITE | 
+                            FILE_SHARE_DELETE,
+                            NULL,
+                            OPEN_EXISTING,
+                            FILE_FLAG_BACKUP_SEMANTICS,
```

Comment:
This is to permit opening a handle on a directory, I assume? Could use a comment explaining that.